python/lsst/afw/image/wcs.cc

```diff
@@ -55,7 +59,15 @@ PYBIND11_PLUGIN(_wcs) {
     mod.def("makeWcs", (PTR(Wcs) (*)(lsst::afw::coord::Coord const &, lsst::afw::geom::Point2D const &,
             double, double, double, double)) makeWcs);
 
-    py::class_<Wcs, std::shared_ptr<Wcs>, lsst::afw::table::io::Persistable, PersistableFacade<Wcs>> cls(mod, "Wcs");
+    py::class_<Wcs,
+               std::shared_ptr<Wcs>,
+               lsst::daf::base::Citizen,
+               lsst::afw::table::io::Persistable,
+               PersistableFacade<Wcs>> cls(mod, "Wcs");
+
+    py::class_<XYTransformFromWcsPair,
+               std::shared_ptr<XYTransformFromWcsPair>,
+               lsst::afw::geom::XYTransform> clsXYTranFromWcs(mod, "XYTransformFromWcsPair");
```

Comment:
I'm going to push back a bit here. Why can't use the style shown in the pybind11 manual and declare the class and follow it with a series of `.def` instead of repeating the name everywhere. This name is so long that it makes the code hard to read if it is repeated for every def.